lib/idp/constants.rb

```diff
@@ -75,7 +75,6 @@ module Vendors
     IAL_MAX = 0
     IAL1 = 1
     IAL2 = 2
-    IAL2_STRICT = 22
```

Comment:
I would consider maybe deprecating this? `# @deprecated` or some other comment?

We have a few `ServiceProviderIdentity` rows that have `ial = 22` in the DB in the int environment

```ruby
ServiceProviderIdentity.where('ial > 2').pluck(:ial).tally
=> {3=>443, 22=>5}
```

On the other hand, int is not prod (and there are no IAL 22 in prod)

Reply:
Since it's not in prod could we downgrade the dev/int users to IAL 2?

Reply:
Meaning just an update for everyone with IAL = 22 to make them IAL 2?